lib/mirage/mirage.mli

```diff
@@ -910,10 +995,7 @@ val merge_git_clients : git_client impl -> git_client impl -> git_client impl
 (** [merge_git_clients a b] is a device that can connect to remote Git
     repositories using either the device [a] or the device [b]. *)
 
-val git_happy_eyeballs :
```

Comment:
could you deprecate the function instead of removing it? That'll give time for our users to migrate progressively.